i18n/zh/docusaurus-plugin-content-docs/version-v1.0.0/design-document/01-runtime-protocol.md

```diff
@@ -43,93 +43,93 @@ public class Header {
 
 + 详解
 
-消息头(header)：类型为Header，Header中有Command字段，用于区分不同的消息类型
+消息头 (header)：类型为 Header，Header 中有 Command 字段，用于区分不同的消息类型
 
-消息体(body)：对于不同的消息类型，body的类型不同
+消息体 (body)：对于不同的消息类型，body 的类型不同
```

Comment:
Sometimes, we don't want to add spaces between Chinese and English punctuation marks. Could you consider adding configuration options to enable or disable these formatting features? Similar to the [Obsidian plugin](https://github.com/Yaozhuwa/easy-typing-obsidian) shown in the image below:

![image](https://github.com/apache/eventmesh-site/assets/41445332/cd4abd09-f31b-4848-86c8-d68b80d38dba)

Reply:
This has disabled and updated the docs.